hal/lib/athena/Notifier.cpp

```diff
@@ -65,7 +65,7 @@ void updateNotifierAlarmInternal(std::shared_ptr<Notifier> notifier_pointer,
   if (triggerTime < closestTrigger) {
     closestTrigger = triggerTime;
     // Simply truncate the hardware trigger time to 32-bit.
-    notifierAlarm->writeTriggerTime((uint32_t)triggerTime, status);
+    notifierAlarm->writeTriggerTime(static_cast<uint32_t>(triggerTime), status);
```

Comment:
These need to stay uint32_t, as timing does require the last bit. They're not user facing